packages/mobile/src/identity/secureSend.ts

```diff
@@ -0,0 +1,161 @@
+import { ErrorMessages } from 'src/app/ErrorMessages'
+import { AddressValidationType, SecureSendPhoneNumberMapping } from 'src/identity/reducer'
+import { Recipient } from 'src/recipients/recipient'
+import Logger from 'src/utils/Logger'
+
+const TAG = 'identity/secureSend'
+
+// Given addresses can be added and deleted we can't rely on changes in length to signal address changes
+// Not sure if address order in array is consistent so not assuming it
+const newAddressesAdded = (oldAddresses: string[], newAddresses: string[]) => {
```

Comment:
This is nbd but the convention we try to keep it is arrow functions in classes (like for handlers) but classic functions for things defined top-level in modules. This facilities recursion and keeps consistency with generators which can't be expressed as lamdbas

Reply:
Gotcha. Will change all the functions in this file and keep that in mind moving forward.